docs/source/conf.py

```diff
@@ -621,6 +621,7 @@ def adds(pth):
     "joblib": ("https://joblib.readthedocs.io/en/stable/", None),
     "scikit-learn": ("https://scikit-learn.org/stable/", None),
     "statsmodels": ("https://www.statsmodels.org/stable/", None),
+    "skbase": ("https://skbase.readthedocs.io/en/latest/", None),
```

Comment:
I think we should link to released version, not to the dev version.

```suggestion
    "skbase": ("https://skbase.readthedocs.io/en/stable/", None),
```

Reply:
agreed